tola/auth_pipeline.py

```diff
@@ -26,6 +28,20 @@ def user_to_tola(backend, user, response, *args, **kwargs):
         userprofile.email = response.get('emails["value"]')
         userprofile.save()
 
+        generated_pass = '%032x' % random.getrandbits(128)
```

Comment:
What is `%032x`?

Reply:
Formatting string. Have a look at the documentation. https://docs.python.org/2/library/string.html